### PR TITLE
Chunk deactivate_expired_reasons command

### PR DESCRIPTION
**Pull request by rubickcz**

### CODE REVIEW

File: .github/workflows/django.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Django CI
```
Comment (rubickcz) [inline .github/workflows/django.yml:1]: The library has tests, it's a pity it does not have CI set up. I fixed it.

File: setup.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def read(fname):
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     ],
     install_requires=[
-        'django>=2.2',
+        'django>=2.2,<=3.2',
```
Comment (rubickcz) [inline setup.py:42]: Latest version of Django is not supported.

File: .github/workflows/django.yml (hunk 1 above)
Comment (matllubos) [reply .github/workflows/django.yml:1]: thanks, maybe fail with travis

File: gdpr/management/commands/deactivate_expired_reasons.py [hunk 3]
```diff
@@ -1,18 +1,40 @@
...
 
 from gdpr.models import LegalReason
 
 from chamber.utils.tqdm import tqdm
 
+from django.conf import settings
 from django.db import transaction
 
 
+logger = logging.getLogger(getattr(settings, 'GDPR_DEACTIVATE_EXPIRED_REASONS_LOGGER', __name__))
```
Comment (matllubos) [inline gdpr/management/commands/deactivate_expired_reasons.py:13]: cool

File: gdpr/management/commands/deactivate_expired_reasons.py [hunk 4]
```diff
@@ -1,18 +1,40 @@
...
+
     def handle(self, *args, **options):
         self.stdout.write('Anonymize expired data of expired legal reasons')
         legal_reason_to_expire_qs = LegalReason.objects.filter_active_and_expired()
-        for legal_reason in tqdm(legal_reason_to_expire_qs.iterator(),
-                                 total=legal_reason_to_expire_qs.count(),
-                                 file=self.stdout):
+        total_number_of_objects = legal_reason_to_expire_qs.count()
+
+        for legal_reason in tqdm(self._slice_queryset(legal_reason_to_expire_qs).iterator(), file=self.stdout):
```
Comment (matllubos) [inline gdpr/management/commands/deactivate_expired_reasons.py]: there is missing total for tqdm. If you use iterator you must set total. 

so

```
sliced_queryset = self._slice_queryset(legal_reason_to_expire_qs)
for legal_reason in tqdm(sliced_queryset.iterator(), total=sliced_queryset.count(), file=self.stdout):
```

**Pull request merged**